contracts/test/vault/oeth-vault.js

```diff
@@ -621,6 +625,20 @@ describe("OETH Vault", function () {
         fixtureWithUser
       );
     });
+    it("Should claim single big request as a whale", async () => {
+      const { oethVault, matt } = fixture;
+
+      await oethVault.connect(matt).requestWithdrawal(oethUnits("30"));
```

Comment:
can you add a post request check that OETH was burned from the user and totalSupply and the vault's `totalValue` also decreased by a comparable amount.